cliqz.cfg

```diff
@@ -50,7 +50,7 @@ lockPref("toolkit.startup.max_resumed_crashes", -1);
 
 // Disable Slow Startup message
 lockPref("browser.slowStartup.notificationDisabled", true);
-
+/*
```

Comment:
why that is commented out?

Reply:
Because with this code it is not possible to start a browser with non existing profile, it show an error "Failed to read configuration file. Please contact your system administrator" and do not launch. I will a separate ticket to find out a solution what to do with this code.